bindings/dotnet/DotOpenDAL/DotOpenDAL.csproj

```diff
@@ -17,10 +17,20 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
 
-    <ItemGroup>
+    <ItemGroup Condition="$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier.StartsWith('osx'))">
+        <None Include="../target/debug/libopendal_dotnet.dylib">
```

Comment:
By the way, can we copy `../target/debug/libopendal_dotnet.*` instead of writing cases for every platform?

Reply:
Let me give a try.

Reply:
It works.